src/sstruct_mv/protos.h

```diff
@@ -30,7 +30,9 @@ HYPRE_Int HYPRE_SStructGridAssemble ( HYPRE_SStructGrid grid );
 HYPRE_Int HYPRE_SStructGridSetPeriodic ( HYPRE_SStructGrid grid , HYPRE_Int part , HYPRE_Int *periodic );
 HYPRE_Int HYPRE_SStructGridSetNumGhost ( HYPRE_SStructGrid grid , HYPRE_Int *num_ghost );
 HYPRE_Int HYPRE_SStructGridPrintGLVis ( HYPRE_SStructGrid grid, const char *meshprefix, HYPRE_Real *trans, HYPRE_Real *origin );
-HYPRE_Int hypre_SStructGridPrint ( hypre_SStructGrid *grid , const char *filename );
+HYPRE_Int HYPRE_SStructGridProjectBox ( HYPRE_SStructGrid grid , HYPRE_Int *ilower , HYPRE_Int *iupper , HYPRE_Int *origin , HYPRE_Int *stride );
```

Comment:
I know we have this in a number of places in hypre, but the 'protos.h' file should not contain the prototypes for the user-level interface routines.  I don't know why we have these here.  It's totally possible that I was even the one who added them, but if I did, I shouldn't have. :)